### DISCUSSION TITLE
【不具合】編集リクエストがエラーになる

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#944] [要望・アイデア・不具合]
長めの記事に編集リクエストしたところ、以下のようなエラーが表示されました。
「サーバとの接続に失敗しました。ネットワーク環境をお確かめの上、再度お試しください。」
![image](https://github.com/user-attachments/assets/6e1bebe3-2158-446c-89a5-90fc262339f1)

何度かリトライしましたが変化ありませんでした。
文字数上限に近くて編集すると文字数オーバーするような状況な気もします。もしそうなら分かりやすいエラーメッセージにしていただけると助かります。
調査していただけますでしょうか。

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご報告ありがとうございます。

開発者コンソールにエラーが表示されているかもしれないので、可能でしたら以下のようなスクリーンショットをご提供いただけないでしょうか🙏

![image](https://github.com/user-attachments/assets/feccfe6d-6046-4bad-93a1-9884bdee1d28)


また、以下も合わせてご確認いただけると幸いです🙇

- セキュリティソフトの設定やネットワーク機器の確認
- 利用しているネットワークを変更しての接続 (別の Wifi への接続など)

お手数をおかけしますが、よろしくお願いします。

**Reply by shiracamus** [reply]
セキュリティソフトは Windows11 の Windows Defender のみです。
ブラウザは Firefox 133.0.3 (64 ビット) です。広告ブロックはオフです。
インターネットルータのログを確認してみたところ、polyfill.ioへの通信がブロックされていましたが、関係あるでしょうか？

![image](https://github.com/user-attachments/assets/40448596-103b-40e1-a193-2e605bac5dc3)

**Reply by shiracamus** [reply]
書き忘れましたが、短い記事に対しては編集リクエストできています。

**Reply by tanaka-asami415** [reply]
@shiracamus 
ご返信ありがとうございます！

この度はご迷惑をおかけして申し訳ありません。502 が出ているので、 Qiita のサーバー側の問題のようです🙇
おそらくですが、データが大きすぎてサーバーがタイムアウトを起こしている可能性がありそうです。

サーバーがタイムアウトを起こしている場合は、今回のメッセージが出るのは正しくないため、エラーメッセージの修正を行い、問題に関しても調査を行います。
追記でいただいた通り、記事が長過ぎることが原因の可能性が高そうです🙇

**Reply by shiracamus** [reply]
今試して見たところ、長い記事への編集リクエストができました。
サーバ負荷が低かったのでしょうかね。

**Reply by ohakutsu** [reply]
@shiracamus 
できたようであればよかったです！
エラーメッセージの修正も対応予定ですので、お待ちいただけると幸いです。
サーバー負荷については、調べてみたのですが断言できない状態でした…🙏
